webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ExecutionBatch.scala

```diff
@@ -746,12 +746,12 @@ final case class ContextForNoAnswer(
             )
         }
 
-      case NoReportInInterval(expectedConfig, _) =>
+      case NoReportInInterval(expectedConfig, expirationTime) =>
         ComplianceDebugLogger.node(nodeId).trace(s"Node didn't received reports recently, status depend of the compliance mode and previous report status")
         buildRuleNodeStatusReport(
             //these reports don't really expires - without change, it will
             //always be the same.
-            MergeInfo(nodeId, None, Some(expectedConfig.nodeConfigId), END_OF_TIME)
+            MergeInfo(nodeId, None, Some(expectedConfig.nodeConfigId), expirationTime)
```

Comment:
that seems to contradict the comment just above, what changed in the logic that implies that change in behaviour?

Reply:
ha, indeed the comment is wrong